src/common.rs

```diff
@@ -233,6 +234,37 @@ impl Config {
         self.target_rustcflags = Some(flags);
     }
 
+    /// Remove rmeta files from library path directories
+    pub fn clean_rmeta(&self) {
+        let varname = if cfg!(windows) {
+            "PATH"
+        } else if cfg!(target_os = "macos") {
+            "DYLD_LIBRARY_PATH"
+        } else {
+            "LD_LIBRARY_PATH"
+        };
+
+        let lib_paths = env::var(varname).unwrap_or_else(|e| {
+            panic!(
```

Comment:
Could this be one line? I *really* don't like how rustfmt splits this on five lines...

Reply:
I will restore `link_deps`'s formatting.

Reply:
Ok, thanks :-)